CMakeLists.txt

```diff
@@ -1576,8 +1576,8 @@ if(QTKEYCHAIN)
     message(FATAL_ERROR "Secure credential storage support requires the Qt5::Keychain component.")
   endif()
   target_compile_definitions(mixxx-lib PUBLIC __QTKEYCHAIN__)
-  target_link_libraries(mixxx-lib PUBLIC ${Qt5Keychain_LIBRARIES})
-  target_include_directories(mixxx-lib PUBLIC ${Qt5Keychain_INCLUDE_DIRECTORIES})
+  target_link_libraries(mixxx-lib PUBLIC ${QTKEYCHAIN_LIBRARIES})
+  target_include_directories(mixxx-lib PUBLIC ${QTKEYCHAIN_INCLUDE_DIRECTORIES})
```

Comment:
According to `Qt5KeychainConfig.cmake` this should be
`target_include_directories(mixxx-lib PUBLIC ${QTKEYCHAIN_INCLUDE_DIRS})`.

Reply:
You're right, I didn't get enough sleep last night ;-)